src/web/templates/admin/players/players_diff.html

```diff
@@ -2,11 +2,7 @@
     <li
         {% if data_source.is_available %}
             class="dropdown-item text-start cursor-pointer"
-            hx-get="{{ url_for(
-                'admin-event-players-diff-modal',
-                event_uniq_id=admin_event.uniq_id,
-                data_source_id=data_source.id
-            ) }}"
+            hx-get="{{ url_for('admin-event-players-diff-modal', event_uniq_id=admin_event.uniq_id, data_source_id=data_source.id) }}"
```

Comment:
Another case where the formatting changes.  I'm convinced that we don't all have teh same line length settings in MyPy, or something like that.